packages/video_player/video_player/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: Flutter plugin for displaying inline video with other Flutter
 # 0.10.y+z is compatible with 1.0.0, if you land a breaking change bump
 # the version to 2.0.0.
 # See more details: https://github.com/flutter/flutter/wiki/Package-migration-to-1.0.0
-version: 0.10.11+2
+version: 0.10.12
```

Comment:
I think we need to depend on the newly published platform interface plugin here?